docs/start/install.md

```diff
@@ -17,46 +19,46 @@ You can install the WasmEdge Runtime on any generic Linux and MacOS platforms. I
 
 ### Generic Linux and MacOS
 
-The easiest way to install WasmEdge is to run the following command. Your system should have `git` and `curl` as prerequisites.
+Your system should have `git` and `curl` as prerequisites. The easiest way to install WasmEdge is to run the following command:
 
 ```bash
 curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash
 ```
 
-Run the following command to make the installed binary available in the current session.
+Run the following command to make the installed binary available in the current session:
 
 ```bash
 source $HOME/.wasmedge/env
 ```
 
 #### Install for all users
 
-WasmEdge is installed in the `$HOME/.wasmedge` directory by default. You can install it into a system directory, such as `/usr/local` to make it available to all users. To specify an install directory, run the `install.sh` script with the `-p` flag. You will need to run the following commands as the `root` user or `sudo` since they are written write into system directories.
+WasmEdge is installed in the `$HOME/.wasmedge` directory by default. You can install it into a system directory such as `/usr/local` to make it available to all users. To specify an install directory, run the `install.sh` script with the `-p` flag. You will need to run the following commands as the `root` user or `sudo` since they are written into the system directories:
 
 ```bash
 curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- -p /usr/local
 ```
 
 #### Install a specific version of WasmEdge
 
-The WasmEdge installer script will install the latest official release by default. You could install a specific version of WasmEdge, including pre-releases or old releases by passing the `-v` argument to the installer script. Here is an example.
+The WasmEdge installer script will install the latest official release by default. You could install a specific version of WasmEdge, including pre-releases or old releases by passing the `-v` argument to the installer script. Here is an example:
 
 ```bash
 VERSION={{ wasmedge_version }}
 curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- -v $VERSION
 ```
 
-Suppose you are interested in the latest builds from the `HEAD` of the `master` branch, which is basically WasmEdge's nightly builds. In that case, you can download the release package directly from our Github Action's CI artifact. [Here is an example](https://github.com/WasmEdge/WasmEdge/actions/runs/2969775464#artifacts).
+If you are interested in the latest builds from the `HEAD` of the `master` branch, which is basically WasmEdge's nightly builds,you can download the release package directly from our Github Action's CI artifact. [Here is an example](https://github.com/WasmEdge/WasmEdge/actions/runs/2969775464#artifacts).
```

Comment:
Could you please revert these changes as well

Reply:
Noted